src/App.js

```diff
@@ -1,12 +1,54 @@
+import { useEffect, useState } from 'react';
 import './App.css';
 import Calculator from './components/Calculator';
+import Header from './components/Header';
+import Qoute from './components/Qoute';
+import Home from './components/Home';
+
+const fetchQuote = async (cb) => {
+  const response = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+    headers: {
+      'X-Api-Key': 'uHRdrDXaqWLwqBrwbx6P9A==Bk4Cvlvw6ePt3sW1',
+    },
+  });
+  const data = await response.json();
+  cb(data);
+};
 
 function App() {
+  const [quotes, setQuotes] = useState([]);
+  const [tab, setTab] = useState();
+  useEffect(() => {
+    setTab('Home');
+    fetchQuote(setQuotes);
```

Comment:
[REQUIRED] Kindly ensure that you have loading and error estates in your code. 
HINT:
```  } catch (error) {
        setHasError(true)
      }
      setIsLoading(false)
    }
```
[OPTIONAL] I think it might be a better idea if you are fetching the api-ninjas data in the `Quotes` component since the data is only used in that component.